samples/v1.0/Scenarios/InputForm.json

```
@@ -3,6 +3,18 @@
 	"type": "AdaptiveCard",
 	"version": "1.0",
 	"body": [
+		{
+			"type": "Container",
+			"verticalContentAlignment": "Center",
+			"style": "warning",
+			"items": [
+				{
+					"type": "TextBlock",
+					"text": "**IMPORTANT:** This card is based on an older schema and doesn't make use of the label property for its Input elements. Because of this, it doesn't properly support accessibility. It's included here for historical purposes. Please see [Input Form With Labels](https://adaptivecards.io/samples/InputFormWithLabels.html) for an updated version of this card.",
+					"wrap": true
+				}
+			]
+		}
```

oops! dropped a `,`:

```suggestion
		},
```

---

@almedina-ms - I think this is the reason why the PR builds were failing in tests :)